code/game/atom/atom_tool_acts.dm

```diff
@@ -0,0 +1,165 @@
+/**
+ * ## Item interaction
+ *
+ * Handles non-combat iteractions of a tool on this atom,
+ * such as using a tool on a wall to deconstruct it,
+ * or scanning someone with a health analyzer
+ */
+/atom/proc/base_item_interaction(mob/living/user, obj/item/tool, list/modifiers)
+	SHOULD_CALL_PARENT(TRUE)
+	PROTECTED_PROC(TRUE)
+
+	// We do not have a combat mode or secondary actions like /tg/, so instead
+	// I'm unilaterally deciding it here: If you are not on harm intent, tool
+	// interactions are not attacks. Shit like the autolathe accepting
+	// screwdrivers on harm intent is unintuitive and needs to go away, and there
```

Comment:
What do you plan on replacing these with? As un-intuitive as it is, being completely unable to recycle screwdrivers seems even less intuitive.